packages/aws-cdk-lib/aws-rds/test/cluster.test.ts

```diff
@@ -170,6 +171,26 @@ describe('cluster new api', () => {
       const stack = testStack();
       const vpc = new ec2.Vpc(stack, 'VPC');
 
+      // WHEN
+      new DatabaseCluster(stack, 'Cluster', {
+        engine: DatabaseClusterEngine.AURORA_MYSQL,
+        vpc,
+        clusterScalabilityType: ClusterScalabilityType.STANDARD,
+        writer: ClusterInstance.serverlessV2('writer'),
+      });
+
+      // THEN
+      const template = Template.fromStack(stack);
+      template.hasResourceProperties('AWS::RDS::DBCluster', {
+        ClusterScalabilityType: 'standard',
+      });
+    });
+
+    test('cluster scalability option with deprecated misspelling', () => {
+      // GIVEN
+      const stack = testStack();
+      const vpc = new ec2.Vpc(stack, 'VPC');
+
```

Comment:
Duplicated tests where the misspelled **ClusterScailabilityType** was used, and used the correct one. I've left the original tests in place, as the misspelling can still be used.